### PR TITLE
Emphasize that the images are not Supported

### PR DESCRIPTION
**Pull request by felixfontein**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 Container image definitions meant for [ansible-test](https://www.ansible.com/blog/introduction-to-ansible-test) and [Execution Environments](https://www.ansible.com/blog/whats-new-in-ansible-automation-platform-2-automation-execution-environments).
 
 ⚠️ **Please note that this repository is very much a proof of concept and a work in progress at this time.** ⚠️
 
-## Important
+## Important ⚠️
 
 Images provided by this repository are tailored for development, testing and CI purposes.
-They are maintained by the community and are not supported for production use: they can and will break or run out of maintenance.
+**They are maintained by the community and are not supported by RedHat**: they can and will break or run out of maintenance.
```
Comment (dmsimard) [inline README.md:10]:
```suggestion
**They are maintained by the community and are not supported by Red Hat**: they can and will break or run out of maintenance.
```

File: ansible-test/README.md [hunk 2]
```diff
@@ -29,14 +29,15 @@ target docker images and supported python version (choose one):
 
 ansible-test supports specifying custom images instead and the purpose of this repository is to provide suitable images for it.
 
-## Important
+## Important ⚠️
 
 Images provided by this repository are tailored for development, testing and CI purposes.
-They are maintained by the community and are not supported for production use: they can and will break or run out of maintenance.
+**They are maintained by the community and are not supported by RedHat**: they can and will break or run out of maintenance.
```
Comment (dmsimard) [inline ansible-test/README.md:35]:
```suggestion
**They are maintained by the community and are not supported by Red Hat**: they can and will break or run out of maintenance.
```

File: execution-environments/README.md [hunk 3]
```diff
@@ -9,14 +9,15 @@ They can be used by [AWX](https://github.com/ansible/awx), [Automation Controlle
 
 They can be built using [ansible-builder](https://github.com/ansible/ansible-builder/) with either [docker or podman](https://ansible-builder.readthedocs.io/en/latest/usage/#container-runtime) using [execution environment definitions](https://ansible-builder.readthedocs.io/en/latest/definition/) provided by this repository.
 
-## Important
+## Important ⚠️
 
 Images provided by this repository are tailored for development, testing and CI purposes.
-They are maintained by the community and are not supported for production use: they can and will break or run out of maintenance.
+**They are maintained by the community and are not supported by RedHat**: they can and will break or run out of maintenance.
```
Comment (dmsimard) [inline execution-environments/README.md:15]:
```suggestion
**They are maintained by the community and are not supported by Red Hat**: they can and will break or run out of maintenance.
```

**Comment by dmsimard** [comment]
WFM, just minor nits (yes, it's two words with capital letters)

**Comment by felixfontein** [comment]
Sorry, I'm always forgetting about that one :)

**Comment by felixfontein** [comment]
@dmsimard thanks again for reviewing and merging!

**Pull request merged**